pallets/avn-offence-handler/src/lib.rs

```diff
@@ -50,7 +50,6 @@ pub mod pallet {
     }
 
     #[pallet::pallet]
-    #[pallet::generate_store(pub (super) trait Store)]
```

Comment:
This seems like a rebase is needed. I think Thanos removed this and it is already merged to main